src/libasr/codegen/wasm_to_x86.cpp

```diff
@@ -68,6 +69,51 @@ class X86Visitor : public WASMDecoder<X86Visitor>,
         }
     }
 
+    void visit_I32Eqz() {}
+
+    void visit_Unreachable() {}
+
+    void visit_EmtpyBlockType() {}
+
+    bool is_else_stmt_available() {
+        auto offset_ = offset;
+        uint8_t cur_byte = wasm::read_b8(code, offset_);
+        while (cur_byte != 0x0B) {
+            if (cur_byte == 0x05) {
+                cur_byte = wasm::read_b8(code, offset_);
+                if (cur_byte == 0x0B || cur_byte == 0x41) {
```

Comment:
I have a doubt:
Does `wasm` always have an else block (even though else is empty) in the if statement?
![image](https://user-images.githubusercontent.com/74826228/199942923-b9905288-a124-4406-aae6-942636ec77b6.png)
@Shaikh-Ubaid 
If so how Assert is handled in asr_to_wasm?

Reply:
I asked this because, the else block is not empty in the `visit_assert()`, it has `0x41`

Reply:
>Does wasm always have an else block (even though else is empty) in the if statement?

Currently, if there is no `else` block, then in the `wat` generated, we would just have `if` `<some instructions>` `end`.

Reply:
>I asked this because, the else block is not empty in the visit_assert(), it has 0x41

Do you mean `visit_Assert()` in `ASR->WASM` backend?

Reply:
yes